index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="top-section">
+        <div class="top-bar">
+            <div class="time"><img src="/images/Header icons/time.png" alt=""></div>
+            <div><img src="/images/Header icons/Cellular Connection.png" alt=""></div>
+            <div><img src="/images/Header icons/Wifi.png" alt=""></div>
+            <div><img src="/images/Header icons/Capacity.png" alt=""></div>
+        </div>
+        <div class="main-menu">
+            <div class="john">John Doe</div>
+            <div class="union"><img src="images/Header icons/Union.png" alt=""></div>
+        </div>
+    </section>   
+    
+        <section class="hero">
+        <div class="heading">
+            <h1>Hey There. <br>I’m Sajid</h1>
+        </div>
+        <div class="software"><h3>I’am a Software Developer</h3></div>
+        <div class="content">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+don’t hestiate to contact me.</div>
+<footer>
+                <ul class="ul">
+                <li><img src="/images/Footer icons/1.png" alt=""></li>
+                <li><img src="/images/Footer icons/2.png" alt=""></li>
+                <li><img src="/images/Footer icons/3.png" alt=""></li>
+                <li><img src="/images/Footer icons/4.png" alt=""></li>
+                <li><img src="/images/Footer icons/5.png" alt=""></li>
+                
+
+            </ul>
+            <div class="ill"><img src="/images/Footer icons/Illustration.png" alt=""></div>
+</footer>    
```

Comment:
- [ ] Kindly remove this footer tag or change it into a div instead because these icons are not part of the footer yet, but I think you can keep the `ul` being the main container of this icon section.